Assets/Prefabs/VrSystems/VrControllers/UnityXR/UnityXRQuestControls.prefab

```diff
@@ -53,10 +53,7 @@ MonoBehaviour:
   - {fileID: 114000013332822508}
   m_HapticsDurationScale: 0.02
   m_HapticAmplitudeScale: 0.04
-  m_TriggerActivationRange: {x: 0.15, y: 0.8}
```

Comment:
I'm not sure if this was supposed to be removed

Reply:
oh doh!

Reply:
actually - looks legit. it's an old property that gets removed only when you resave a prefab

Reply:
ok, excellent. I thought this might have been one of those things i know Mike spent a lot time tuning around sensitivity